common-requests/use.md

```diff
@@ -1,8 +1,157 @@
+    "architecture": "amd64",
+    "horizon_version": "2.26.12"
+  }
+}
+```
+
+This output will require a little more explanation. 
+Beginning with the first two lines, `id` refers to the ID we assigned to this edge node, and `organization` refers to the ad-hoc group that the device "belongs" to. 
+
+Next, take a look at the `configstate.state` property.  If the value is "configured", then this tells you that the edge node is currently registered for a service.
+
+Last, look at the `configuration.exchange_version` property.  If you do not see a version number as the value, this is most likely because the Agent cannot query the Exchange service for the version, which may mean that your Agent is not connected to the Management Hub.  This will most likely not happen using the all-in-one installation.
+
+## Look at the Agreement formed when your node registered for a service
+
+``` shell
+hzn agreement list
+```
+
+This command will display a JSON array of objects showing one or more active agreements between the edge node and AgBots:
+
+``` json
+[
+  {
+    "name": "Policy for myorg/node1 merged with myorg/policy-ibm.helloworld_1.0.0",
+    "current_agreement_id": "83785680de5d22c874a0f54dc229738a0e744db22b2c0deeb320b9fdf0967138",
+    "consumer_id": "IBM/agbot",
+    "agreement_creation_time": "2020-08-04 16:44:36 -0400 EDT",
+    "agreement_accepted_time": "",
+    "agreement_finalized_time": "",
+    "agreement_execution_start_time": "",
+    "agreement_data_received_time": "",
+    "agreement_protocol": "Basic",
+    "workload_to_run": {
+      "url": "ibm.helloworld",
+      "org": "IBM",
+      "version": "1.0.0",
+      "arch": "amd64"
+    }
+  }
+]
+```
+
+Pay attention to the following properties:
+* `0.agreement_creation_time` - When the agreement was proposed by the AgBot
+* `0.agreement_accepted_time` - When the agreement was accepted by the edge node
+* `0.agreement_finalized_time` - When the agreement was completed
+* `0.agreement_execution_start_time` - When the edge node began downloading the container image(s)
+
+Once you see a value for `agreement_execution_start_time`, you should expect to be able to run `docker ps` to see the workloads starting.
+
+## Other things to try
+
+To see the logs of the sample "helloworld" service installed by default, try `hzn service log -f ibm.helloworld`.
+
+To view all of the steps logged in the process of negotiating an agreement, type `hzn eventlog list`.
+
+And to view the node's policy that triggered deployment of the "helloworld" service, try `hzn policy list`.
+
+## One last thing ...
+
+We've been using the CLI to communicate with the Anax agent.  There's a whole other set of commands to run that have you communicating with the Exchange.  To get those working, you'll need to set two environment variables.  To do so, you'll need one important bit of information from the all-in-one installation summary message.  Hopefully you saved that information.  It should look something like this:
```

Comment:
> To use the exchange communication commands, set up two environment variables. 

**Prerequisite**

You need this information from the all-in-one installation summary message that you saved in (tell the what they should have saved and where to find it AND we need to tell them in the orig step what they were supposed to save):